src/lib.rs

```diff
@@ -26,6 +26,7 @@ extern crate termion;
 
 #[cfg(windows)]
 use winapi::shared::minwindef::DWORD;
+use winapi::shared::ntdef::WCHAR;
```

Comment:
Missing `#[cfg(windows)]` for `winapi::shared::ntdef::WCHAR`

Reply:
Oops, thx!